From dc622cf48bed312045d6a7762d166803db28dff3 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Sun, 28 Jul 2019 13:27:46 +0300
Subject: [PATCH 2/6] armv8: lx2160acex: misc hacks to get the sources built

those hacks will be sorted out nicer in the future and this patch will
not be needed anymore

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 arch/arm/cpu/armv8/fsl-layerscape/Kconfig | 1 -
 drivers/mmc/fsl_esdhc.c                   | 2 +-
 include/configs/lx2160a_common.h          | 4 ++--
 3 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/arm/cpu/armv8/fsl-layerscape/Kconfig b/arch/arm/cpu/armv8/fsl-layerscape/Kconfig
index 8c47d81f7f..9236be05c8 100644
--- a/arch/arm/cpu/armv8/fsl-layerscape/Kconfig
+++ b/arch/arm/cpu/armv8/fsl-layerscape/Kconfig
@@ -225,7 +225,6 @@ config ARCH_LX2160A
 	select SYS_FSL_DDR_LE
 	select SYS_FSL_DDR_VER_50
 	select SYS_FSL_EC1
-	select SYS_FSL_EC2
 	select SYS_FSL_ERRATUM_A050106
 	select SYS_FSL_HAS_RGMII
 	select SYS_FSL_HAS_SEC
diff --git a/drivers/mmc/fsl_esdhc.c b/drivers/mmc/fsl_esdhc.c
index 53277c9216..1ec74bd6ba 100644
--- a/drivers/mmc/fsl_esdhc.c
+++ b/drivers/mmc/fsl_esdhc.c
@@ -1497,7 +1497,7 @@ static int fsl_esdhc_probe(struct udevice *dev)
 #endif
 	}
 
-	priv->wp_enable = 1;
+	priv->wp_enable = 0;
 
 #ifdef CONFIG_DM_GPIO
 	ret = gpio_request_by_name(dev, "wp-gpios", 0, &priv->wp_gpio,
-- 
2.17.1

